_opcodes/op023.html

```diff
@@ -16,6 +17,15 @@
 <br/>1   Flat Value Modifier -> Morale = 'Statistic Modifier' value
 <br/>2   Percentage Modifier -> Morale = (Morale * 'Statistic Modifier' value) / 100
 
+<br/>Known values for 'Mode' are:
+<br/>0   BG2 mode -> It's hardcoded to fill the following values:
+<ul>
+  <li>param #1 = 10</li>
+  <li>param #2 = 1</li>
+  <li>m_done = 1 (internal field that immediately removes the effect from creature, no matter duration)</li>
+</ul><br/>In addition, it checks if the creature is berserking, and if it is, it returns without doing anything.
+<br/>1   BG1 mode -> Default (functions as expected)
+
```

Comment:
You can still simplify this as noted earlier (come on, think of the reader), plus it is now incorrect. Mode will be 0 by default, but this distinction only holds for ee, maybe just iwdee. All the other games are not affected. So it needs an EE disclaimer at the start.